targeted/SQLMapCommandGenerator.js

```diff
@@ -0,0 +1,37 @@
+//it will generate and copy sqlmap command based on the request
+//released under the Apache v2.0 licence.
+//You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//author: @juliosmelo
+
+
+function invokeWith(msg) {
+	var string = "sqlmap --url '"+msg.getRequestHeader().getURI().toString()+"' \\\n";
+	var header = msg.getRequestHeader().getHeadersAsString();
+	header = header.split(msg.getRequestHeader().getLineDelimiter());
+
+	for(var i=0;i<header.length;i++){
+		//blacklisting Host (other blacklisting should also specify here
+		var keyval = header[i].split(":");
+		if(keyval[0].trim() != "Host"){
+			string += " -H '"+header[i].trim()+"' ";
```

Comment:
` '"+header[i].trim()+"' ";` could be done after the if/else, it's being done in both branches.

Reply:
Hey, @thc202  thanks for the feedback. It's done

Reply:
The changes don't seem to be correct, as is now it's sending `Host: Host: localhost:9000`. The else should be added/kept to either add `--host` or `-H`, not both when the `Host` header. Or just remove the extra Host header handling, `sqlmap` send the host header also when specified with `-H`.

Reply:
Removed the host header handling.